cmd/goplay/main.go

```diff
@@ -34,10 +34,13 @@ func main() {
 }
 `
 
+var cwd string
```

Comment:
Don't use a global. Pass `cwd` as an extra argument.

Actually, you don't need this if you just `os.Chdir` in `main` function.